code/style.css

```diff
@@ -1,4 +1,266 @@
-html {
-  background: #0025ff;
-  color: #fff;
+*   {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
+
+html, body {
+  width: 100%;
+  height: 100%;
+}
+
+body      {
+  background-color: #e9e9e9;
+  font-family: 'Courier New', Courier, monospace, sans-serif;
+  font-size: 1rem;
+  color: #444444 
+}
+
+
+/* HEADER */
+
+header  {
+  height: 80px; 
+  display: flex;
+  align-items: center;
+  padding: 0 16px;
+}
+
+.hamburger  {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 24px;
+}
+
+.hamburger  div {
+  width: 30px;
+  height: 5px;
+  background: black;
+  border-radius: 30%;
+}
+
+.logo {
+  width: 50px;
+  height: 50px;
+  margin: 0;
+  padding: 5px;
+  border-radius:50%;
+}
+
+nav a {
+  color: black;
+  text-decoration: none;
+  margin-left: 16px;
+  grid-area: nav;
+}
+
+
+/* NEWS CONTAINER */
+
+.news-container {
+  padding: 100px 16px;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 1000px;
+  margin: 0 auto; 
+}
+
+.news-container img {
+  width: 100%;
+  border-radius: 50%;
+}
+
+.news-container div {
+  text-align: center;
+}
+
+
+/* SMALL POSTS */
+
+.post-wrap  {
+  min-width: 300px;
+  max-width: 380px;
+  height: 420px;
+  perspective: 1000px;
+  flex: 1;
+  margin: 0 15px 30px;
+}
+
+.post     {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  transition: transform 1s;
+  transform-style: preserve-3d;
+}
+
+.post-front, .post-back {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background-color: white;
+  backface-visibility: hidden;
+  border-radius: 10px;
+  box-shadow: 0 4px 6px -1px rgba(0,0,0,0.1);
+}
+
+.post-back   {
+  transform: rotateY(180deg);
+}
```

Comment:
These are some really nice features, great job 👏

Reply:
Thank you.